Lab_1/.gitignore

```diff
@@ -0,0 +1,2 @@
+build/
```

Comment:
Please, remove this file. Build folder is inherits from develop, app inside better to remove by make clean before push.